src/main/docs/guide/introduction/whatsNew.adoc

```diff
@@ -1,6 +1,10 @@
 //Micronaut {version} includes the following changes:
 == 4.0.0
 
+=== Kotlin applications via KSP
```

Comment:
Change to:

```
=== Experimental Support for Kotlin Symbol Processing (KSP)
```

Reply:
close via https://github.com/micronaut-projects/micronaut-core/pull/8786/commits/13dd300dc4b77838c4ef980d933ab5bd759c513a